### PR TITLE
Added DiffNow

### LINKED ISSUE #177: DiffNow
https://www.diffnow.com/
Advanced, online and free File Difference checking, with options like Upload file, compare files by their links or Text Input.
It is better than existing entry in *Text Based Tools* as the prev. one. doesn't give you options for uploading files and when it comes to comparing large files (test cases), we better upload one file.

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #177 #190 

**https://www.diffnow.com/**

**Advanced, online and free File Difference checking, with options like Upload file, compare files by their links or Text Input.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

**Comment by aviaryan** [comment]
@aashutoshrathi Is the site working as expected now?

**Comment by aashutoshrathi** [comment]
Yes

**Comment by aviaryan** [comment]
@aashutoshrathi Please resolve conflicts.

File: README.md [hunk 1]
```diff
@@ -283,9 +283,11 @@ See [plotting graphs examples](https://www.wolframalpha.com/examples/PlottingAnd
 * [PolishMyWriting](http://www.polishmywriting.com/) - Paste a text and get feedback on your writing.
 * [Wordle](http://www.wordle.net/) - A toy to generate 'Word Clouds' form a given piece of text.
 * [tYp3r](http://typ3r.aavi.me) - dA aNn0Y1Ng t3Xt g3NeRa7or (The annoying text generator :-P).
+* [DiffNow](https://www.diffnow.com/) - Advanced, online and free File Difference checking, with options like Upload file, compare files by their links or Text Input.
 * [Telegraph](http://telegra.ph/) - Minimalist publishing tool that supports richly formatted text.
 
 
+
```
Comment (aviaryan) [inline README.md]: Extra line added. This doesn't correspond with the file's conventions.

**Comment by aviaryan** [comment]
Thanks for the update, @aashutoshrathi.

**Pull request merged**